munit/js/src/main/scala/munit/internal/Path.scala

```diff
@@ -1,70 +1,63 @@
 package munit.internal
 
-import java.io.File
 import java.net.URI
-import java.nio.file.WatchEvent.{Kind, Modifier}
-import java.nio.file._
 import java.util
 
 import scala.collection.JavaConverters._
 
 // Rough implementation of java.nio.Path, should work similarly for the happy
 // path but has undefined behavior for error handling.
-case class NodeNIOPath(filename: String) extends Path {
+case class Path(filename: String) {
```

Comment:
Let's call this `MUnitPath` instead of `Path`. Or something else if you have suggestions. I don't like it when libraries use generic names for internal classes because it pollutes the namespace when you're auto-completing symbols.

Reply:
Makes sense, done.